PublishParasoftResults/CoverageReportService.ts

```diff
@@ -80,9 +80,9 @@ export class CoverageReportService {
                 baseCoverage = this.mergeCoberturaCoverage(lodash.cloneDeep(baseCoverage), reportToMerge);
             } catch (error) {
                 if (error instanceof Error) {
-                    tl.warning(`Skipped merging Cobertura report '${reportPaths[i]}': ${error.message}`);
+                    tl.warning(`Coverage data in report '${reportPaths[i]}' was not merged due to ${error.message}`);
                 } else {
-                    tl.warning(`Skipped merging Cobertura report: ${reportPaths[i]}`);
+                    tl.warning(`Coverage data in report '${reportPaths[i]}' was not merged`);
```

Comment:
I think it's worth printing out some details related to the error. Since most standard exceptions are instances of `Error`, we can consider printing `JSON.stringify(error)` **in the next line** to see details about the error.